### ISSUE TITLE
'npm test' fails on Python 3 in vendor/js/wmd/jsmin.py 

### ISSUE THREAD

**Issue by cclauss** [internetarchive/openlibrary#1695]
With the mods from #1693, Travis Python 3 [__npm test__](https://travis-ci.org/internetarchive/openlibrary/jobs/465262458#L918-L935) fails...
```
bash static/js/vendor.jsh > static/build/vendor.js
Traceback (most recent call last):
  File "static/js/../../vendor/js/wmd/jsmin.py", line 33, in <module>
    from StringIO import StringIO
ModuleNotFoundError: No module named 'StringIO'
```

Can we revendor to get Python 3 compatible versions?  https://pypi.org/project/jsmin

**Comment by tfmorris** [comment]
wmd hasn't been updated in 10 years and the [ReadMe](https://github.com/derobins/wmd) says "Please use PageDown instead". It looks like [PageDown](https://github.com/ujifgc/pagedown) hasn't been updated in 4 years, so there could be better alternatives.

jsmin.py wasn't/isn't part of the wmd repo, but was added by Anand in 2009.

It looks like the simplest fix is just to patch jsmin.py (It's unrelated to the current pypi jsmin, as far as I can tell).

**Comment by cclauss** [comment]
The jsmin versions (including the PyPI version) all seem to share a similar genealogy of [Crockford --> Even --> St.Germain](https://github.com/tikitu/jsmin/blob/master/jsmin/__main__.py#L3-L8) and the reason that I like the [PyPI version](https://pypi.org/project/jsmin) is because:
1. "Recent" updates have been made:
    * https://raw.githubusercontent.com/tikitu/jsmin/master/CHANGELOG.txt
2. Tests pass on all current versions of Python:
    * https://raw.githubusercontent.com/tikitu/jsmin/master/.travis.yml

**Comment by cclauss** [comment]
Well done @tfmorris closed in internetarchive/wmd#2.
